packages/serverless-orchestration/src/ServerlessSpoke.js

```diff
@@ -105,18 +120,19 @@ function _execShellCommand(cmd, inputEnv, strategyRunnerSpoke = false) {
 // Format stdout outputs. Turns all logs generated while running the script into an array of Json objects.
 function _stripExecStdout(output, strategyRunnerSpoke = false) {
```

Comment:
note how this function was changed: there is no mapping going on here. `logsArray` is the full log object generated within the spoke, returned as part of the `stdout`. depending on the log produced we can sepect what parts to emit above.